### ISSUE TITLE
Import items from autogenerated crud file

### ISSUE THREAD

**Issue by phillipmadsen** [caffeinated/menus#64]
Is there a way to import the necessary items from a different file.  here is an example.

public function handle($request, Closure $next)
    {
        @include('AdminMenu.php');
        return $next($request);
    }

and inside adminmenu.php I  have items that are autogenerated from my crud builder script. 

Menu::make('ecommerce', function($menu) {
$menu->add('Users', 'admin/users')->prepend('<i class="fa fa-users"></i>')->active('admin/*');;
$menu->users->add('Add User', 'admin/users/create');
});

**Comment by ghost** [comment]
Yes, but you need to work with Menu::make() and Menu::get() methods.
